cmd/vic-machine/configure/configure.go

```diff
@@ -87,7 +88,8 @@ func (c *Configure) Flags() []cli.Flag {
 	}
 
 	proxies := c.proxies.ProxyFlags(false)
-
+	// after issue #5472 is fixed, change hidden back to false
```

Comment:
It'd be nice to add a `TODO(<your name here>)` before the comment. Often, `git blame` is tough to navigate when code gets moved around.